### PR TITLE
Add Esperanto Translations

### PR DESCRIPTION
**Pull request by darccyy**
Add folder `eo` to `localisation`, containing translations of all `json` files into Esperanto.
These translations are not perfect, but they should be good enough.

Notes:
Acronyms such as 'NSFW' were left the same.
I assumed these translations were accurate:
-   premium = premio
-   bot = roboto
-   Discord Server = Diskorda Servilo
-   NSFW = NSFW (otherwise NSPL for Ne Sekura Por Laborejo)
-   doujin (fandom) = fervolularo (aro de fervoluloj)
-   slash command = oblikva ordono
-   tag = etikedo
-   pong (return of ping command) = respondo (responce)
-   run (a command) = ordoni
-   memory (RAM) = memorilo
-   bookmark = legosignoj (plural)

`.vscode/settings.json` was also added for for tab size formatting.

`localisation/eo/ESPERANTO_NOTES.md` can be deleted, it is just a mirror of this PR for future reference.

### CODE REVIEW

**Comment by darccyy** [comment]
Hope this is helpful :)

**Comment by reinhello** [comment]
Thank you for your contribution! I don't speak or know the language but I think the translations are all good, will add them soon in [Crowdin](https://crowdin.com/project/nreader) for everyone else to contribute/proofread.

File: localisation/eo/ESPERANTO_NOTES.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+-   bot = roboto
+-   Discord Server = Diskorda Servilo
+-   NSFW = NSFW (otherwise NSPL for Ne Sekura Por Laborejo)
+-   doujin (fandom) = fervolularo (aro de fervoluloj)
+-   slash command = oblikva ordono
+-   tag = etikedo
+-   pong (return of ping command) = respondo (responce)
+-   run (a command) = ordoni
+-   memory (RAM) = memorilo
+-   bookmark = legosignoj (plural)
```
Comment (reinhello) [inline localisation/eo/ESPERANTO_NOTES.md:19]: You may delete this file and everything else should be good.

**Review by reinhello — CHANGES_REQUESTED** [review]

**Comment by reinhello** [comment]
One important thing to notice is that you'll also need to edit a few of the framework's code in order to fully support this language. Feel free to join the [Discord server](https://discord.gg/b7AW2Zkcsw) for devs discussion if you like to.

**Review by reinhello — APPROVED** [review]

**Comment by darccyy** [comment]
Thanks!

**Pull request merged**